internal/trace/transport.go

```diff
@@ -22,11 +22,12 @@ import (
 	"sync/atomic"
 )
 
+var count uint64
```

Comment:
Since `count` is a global variable now, how about naming it as `requestCount`?
Meanwhile, could you write documentation for it?

Reply:
renamed & doc added